lib/index.js

```diff
@@ -23,12 +23,20 @@ var getOpts = function (searchQuery, options) {
       q: searchQuery,
       limit: options.limit || DEFAULT_LIMIT
     },
-    auth: {
-      username: options.user,
-      password: options.pass
-    },
     debug: !!options.debug
   };
+
+  if (options.token) {
+    opts.headers = {
+      'Authorization': 'Bearer ' + options.token
```

Comment:
This can be `opts.auth.bearer = options.token` which simplifies this API a bunch.

https://github.com/request/request#http-authentication